colin/core/ruleset/ruleset.py

```diff
@@ -118,8 +118,12 @@ def get_checks_path():
 
     :return: str (absolute path of directory with checks)
     """
-    rel_path = os.path.join(os.pardir, os.pardir, os.pardir, "checks")
-    return os.path.abspath(os.path.join(__file__, rel_path))
+    out_path = os.environ.get("CHECKS") or os.path.join(__file__,
```

Comment:
could you please add this as a new option to the CLI?

Reply:
`click` actually supports this via e.g. [`envvar='COLIN_RULESET'`](https://github.com/user-cont/colin/blob/master/colin/cli/colin.py#L47)

Reply:
I have no idea how to do it in the best way, because of no knowledges, how `click` works. Because I need to have possibibity also set it without any cli interface. for my purpose.

In fact I can duplicate it in `click` and then have it also in this `get_check_path function()` is that right way?